### ISSUE TITLE
self-contained

### ISSUE THREAD

**Issue by tonisimakov99** [dotnet/dotnet-monitor#4683]
Is it possible to support a self-contained build?

**Comment by jander-msft** [comment]
Yes, this is possible and we have some aspiration to support this, but we have a few work items to get through before we can officially support this. I'll try to create a separate work item with the known work for this effort.
